plugins/org.python.pydev/pysrc/runfiles.py

```diff
@@ -72,10 +72,13 @@ def main():
         else:
             raise ImportError()
 
-    except ImportError:
+    except ImportError as e:
```

Comment:
Wow, that's old, I had no idea :( Is that documented somewhere? Anyway, will rework the commit.